android/scripts/release-sdk.sh

```diff
@@ -107,5 +131,10 @@ if [[ $DO_GIT_TAG == 1 ]]; then
     git tag android-sdk-${SDK_VERSION}
 fi
 
+if [ ${BUILD_MODE} == "dev" ]; then
```

Comment:
This won't be run when the build fails at some previous step. Is that okay?
If not, this could be done in an error trap.
something like
```
cleanup() {
  if [ ${BUILD_MODE} == "dev" ]; then
    ...
  fi
}

trap cleanup ERR

update_version_if_dev_build() {
...
}
...
```